### PR TITLE
Add Service Machine Access policies table

### PR DESCRIPTION
**Pull request by clairbarrass**
Hi Andy
I've put together the Machine Access Policies, as gleaned from the SAFE, as we discussed at Uadmin earlier.
Please could you review, and check I've not totally misunderstood, or got it all wrong!

### CODE REVIEW

File: docs/safe-for-users.md [hunk 1]
```diff
@@ -130,6 +130,22 @@ This can be safely ignored - no changes will be made if the link is not used.
...
+
+| Service  |  Access policy  |
+| -------- | --------------  |
+|	ARCHER		|	Users must have email addresses from specified organisations<br>Users must have a public key registered to use the machine  |
+| archer2-4c	<br>4 Cabinet  |  Users must have a public key registered to use the machine |
+| Bede			|     Only one account per person is allowed |
+| Cirrus		| Users must have email addresses from specified organisations<br>    Users can register a public key to access the machine |
+| CSD3	| Only one account per person is allowed<br>    Users can register a public key to access the machine<br>    Password authentication is not allowed |
+| Kelvin2			|     Only one account per person is allowed<br>    Users can register a public key to access the machine<br>    Password authentication is not allowed |
+| Thomas		|     Only one account per person is allowed<br>    Users can register a public key to access the machine<br>    Password authentication is not allowed |
```
Comment (aturner-epcc) [inline docs/safe-for-users.md]: Just a few additional bits of information that (I think) make the table more useful:

```
| Service  |  Hosting site | Access policy  |
| -------- | --------------  |
| ARCHER | EPCC | Users must have email addresses from specified organisations<br/>Users must have a public key registered to use the machine<br/>Both password and SSH key required for access  |
| ARCHER2 |  EPCC | Users must have an SSH public key registered to use the machine<br/>Both password and SSH key required for access |
| Bede | N8 | Only one account per person is allowed |
| Cirrus | EPCC | Users must have email addresses from specified organisations<br/>Users must register an SSH public key to access the machine<br/>Both password and SSH key required for access |
| CSD3 (Peta4-Skylake, Peta4-KNL, Wilkes2-GPU) | University of Cambridge | Only one account per person is allowed<br/>Users must register an SSH public key to access the machine<br/>SSH key used for access (no password-based access) |
| Kelvin2 | Queen's University Belfast | Only one account per person is allowed<br/>Users must register an SSH public key to access the machine<br/>SSH key used for access (no password-based access) |
| Thomas	| MMM Hub | Only one account per person is allowed<br/>Users must register an SSH public key to access the machine<br/>SSH key used for access (no password-based access) |
```

**Review by aturner-epcc — CHANGES_REQUESTED** [review]
I have added a bit more information in the review comment that should be able to be copy and pasted into source and committed again (I could not work out how to do the modifications myself, sorry).

File: docs/safe-for-users.md (hunk 1 above)
Comment (clairbarrass) [reply docs/safe-for-users.md]: Have made the changes and raised PR once more

**Review by aturner-epcc — APPROVED** [review]
LGTM, Merging

**Pull request merged**